src/NzbDrone.Core/MediaFiles/ScriptImportDecider.cs

```diff
@@ -118,12 +173,27 @@ public ScriptImportDecision TryImport(string sourcePath, string destinationFileP
             _logger.Debug("Executed external script: {0} - Status: {1}", _configService.ScriptImportPath, processOutput.ExitCode);
             _logger.Debug("Script Output: \r\n{0}", string.Join("\r\n", processOutput.Lines));
 
-            switch (processOutput.ExitCode)
+            var scriptImportInfo = ProcessStdout(processOutput.Lines, destinationFilePath);
+
+            var mediaFile = scriptImportInfo.MediaFile;
+            localEpisode.PossibleExtraFiles = scriptImportInfo.PossibleExtraFiles;
+
+            episodeFile.RelativePath = series.Path.GetRelativePath(mediaFile);
+            episodeFile.Path = mediaFile;
+
+            if ((processOutput.ExitCode & 0x4) == 0x4)
+            {
+                localEpisode.ShouldImportExtras = true;
+            }
+
+            switch (processOutput.ExitCode & 0x3)
```

Comment:
Should be able to achieve all this without mixing in the bit-wise operator along with an if and a switch which will make it easier to read and maintain.

Reply:
Personally I don't really like my new alternative any better...

Reply:
Did you forget to push these changes up?

Reply:
I did indeed...... Fixed.